### PR TITLE
Update Windows download to direct link

### PR DESCRIPTION
**Pull request by csabella**
The `releases` link for Windows currently goes to a page that requires clicking on a link to go to the correct Downloads page.  This change will link directly to the downloads page.

### CODE REVIEW

**Comment by csabella** [comment]
@willingc, would you be able to review this?  Thanks!

File: Doc/using/windows.rst [hunk 1]
```diff
@@ -18,7 +18,7 @@ Installing Python
 Unlike most Unix systems and services, Windows does not include a system
 supported installation of Python. To make Python available, the CPython team
 has compiled Windows installers (MSI packages) with every `release
-<https://www.python.org/download/releases/>`_ for many years. These installers
```
Comment (tirkarthi) [inline Doc/using/windows.rst:21]: Thanks for this since the previous link (https://www.python.org/download/releases/) contains very old releases. As a suggestion since this page is specific to Windows https://www.python.org/downloads/windows/ seems to be a better link in Windows context that has all the releases along with direct links to download Windows installers.

Comment (csabella) [reply Doc/using/windows.rst:21]: @tirkarthi Thanks for the feedback!  Yes, that other link is referenced further down on this same page under 'Other Platforms', so I'm not sure which is better.  There's two ways of looking at it -- if it links to `downloads/windows`, there is a cookie crumb at the top to backup one level to downloads.  But, if it links to `downloads` directly, then there's an obvious choice (the big yellow button) for the most likely version that someone would want and immediately under that button is the link to the `downloads/windows` page.  That's why I choose to link to `downloads` here, but, as I said, I'm not sure which is better.

Comment (tirkarthi) [reply Doc/using/windows.rst:21]: Got it. Thanks for the clarification. I would wait for the reviewer's feedback 👍

**Comment by willingc** [comment]
@csabella Sorry for the delay. Hmm... I'm not quite sure the best link to use. As a user pulling up python.org and looking for a Windows download, I went to the menu Downloads and submenu Windows which landed me on this page https://www.python.org/downloads/windows/ 

I'm not using a Windows system so I don't know if the site would recognize that I am on Windows and direct accordingly. Since the section is talking about the many Windows releases, I guess I would lean to using this link: https://www.python.org/downloads/windows/ 

Thoughts @csabella?

**Comment by csabella** [comment]
@willingc   Thanks!  I hadn't thought about not being able to see the link on a non-Windows system.  Here's a screenshot of what the /downloads page looks like on Windows 10.  So, linking directly to that page would give the big yellow button to download the latest release directly, with the option right under that button to go to the /downloads/windows page to see all the releases.  

image

Having looked at this again now and taking into account the page it's coming from, I think /downloads would be better with the button directing to the latest release.  If a new user was accessing this, I think having too many options is more confusing.

**Comment by willingc** [comment]
@csabella Thanks for the additional detail. How about this: Use the download link to the big yellow button page for current release and use the longer url to the releases page for the previous releases

I'm a bit hesitant when talking about prior releases to assume that folks will see the link below the yellow button since the prompt is about other operating systems.

tl;dr Reword to include current and previous each with a link to the corresponding page.

**Comment by csabella** [comment]
@willingc Thanks for the suggestion!  I think it's cleaner now.

**Review by tirkarthi — APPROVED** [review]
LGTM. Thanks @csabella :)

**Comment by vstinner** [comment]
Honestly, I don't understand well where there are 2 different web pages which seem to contain mostly the same links and info. I cannot say if the change is good or not, I'm confused :-(

**Comment by vstinner** [comment]
Oh. I didn't notice that the current doc points to https://www.python.org/download/releases/ What is this page? Why do we have so many pages to download pages? Maybe this part of python.org should be redesigned :-( As an user, it's *very* confusing to have so many choices...

**Comment by willingc** [comment]
I agree @vstinner. @csabella how about we have this issue be part of the mentored sprint? cc/@mariatta

**Comment by csabella** [comment]
@willingc Sure, I can close this and add it to the list for the sprint.  The hardest part with this one is making sure there is consensus.

**Comment by vstinner** [comment]
> @willingc Sure, I can close this and add it to the list for the sprint. The hardest part with this one is making sure there is consensus.

Is there a sprint on https://github.com/python/pythondotorg?

**Comment by willingc** [comment]
> Is there a sprint on https://github.com/python/pythondotorg?

@vstinner This is for the PyCon Mentored Sprint in Cleveland.

@Mariatta Do you want me to add it?

**Comment by vstinner** [comment]
> @vstinner This is for the PyCon Mentored Sprint in Cleveland.

I know. But I don't understand if @csabella wants to re-propose a similar change on CPython documentation, or if she wants to propose a change on python.org.

**Comment by willingc** [comment]
Ahh...I was thinking @csabella may pair with an attendee at the mentored sprint on this issue :smile:

**Comment by csabella** [comment]
> @Mariatta Do you want me to add it?

I've started a list for stuff that isn't on the bpo yet.  I asked Mariatta if she wanted to use a shared doc.  Not sure if one exists already.

> But I don't understand if @csabella wants to re-propose a similar change on CPython documentation, or if she wants to propose a change on python.org.

I think both will be needed.  The link from the Windows page here needs to be updated, but Victor's concerns about too many landing pages for releases probably also needs to be discussed on python.org, especially if it's confusing to users looking for the right download.  The second one is probably more complicated.

> Ahh...I was thinking @csabella may pair with an attendee at the mentored sprint on this issue
That's a possibility.  As it's my first PyCon, I hope to spend some time with the sprint, but I'm not sure about the details yet.  I think more info will be available once we see how many people sign up and how many issues we find.

**Comment by ned-deily** [comment]
FTR, https://www.python.org/download/releases/ is now a link to legacy releases that were not added to the website release database when the "new" python.org was launched several years ago.  The link to that page should be changed, i.e. it should only be referenced from the "new" canonical download page ("View older releases" link).  That is now https://www.python.org/download/ which provides a list of all (but the oldest legacy releases mentioned above) releases plus a download button tailored based on the browser's platform information; there are also links on that page to automatically-generated platform-specific download pages (https://www.python.org/downloads/windows/, https://www.python.org/downloads/mac-osx/, https://www.python.org/downloads/source/).

**Comment by ned-deily** [comment]
P.S. I did a quick search of `master` and `Doc/using/windows.rst` seems to be the only reference to the outdated URL in the source tree.
